cocos/core/pipeline/pipeline-scene-data.ts

```diff
@@ -61,10 +62,30 @@ export class PipelineSceneData {
     public get fpScale () {
         return PipelineSceneDataPool.get(this._handle, PipelineSceneDataView.FP_SCALE);
     }
-
     public set fpScale (val: number) {
         PipelineSceneDataPool.set(this._handle, PipelineSceneDataView.FP_SCALE, val);
     }
+
+    public get deferredLightingMat () {
+        return this._deferredLightingMat;
+    }
+
+    public set deferredLightingMat (mat: Material) {
+        if (this._deferredLightingMat === mat) return;
+        this._deferredLightingMat = (mat && mat.effectAsset) ? mat : builtinResMgr.get<Material>('builtin-deferred-material');
+        this.updateDeferredPassInfo();
+    }
+
+    public get deferredPostMat () {
+        return this._deferredPostMat;
+    }
+
+    public set deferredPostMat (mat: Material) {
```

Comment:
The above api naming don't follow our rules, Mat should be Material instead

Reply:
> The above api naming don't follow our rules, Mat should be Material instead

Already rename all the Mat.